src/pds/ingress/util/config_util.py

```diff
@@ -45,6 +52,11 @@ def get_config(config_path=None):
             The parser instance containing the contents of the read config.
 
         """
+        global CONFIG
```

Comment:
Could concurrency be an issue here? Should there be a mutex in case multiple threads try to get the `CONFIG`?

Reply:
My intention is to always have the config and logger singleton objects be initialized in the main thread prior to any concurrent activities, so I think this approach should be fine for now. If a mutex is required, I will hash that out while implementing concurrent uploads to S3.